enterprise_access/apps/customer_billing/migrations/0018_merge_20251105_0545.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 4.2.25 on 2025-11-05 05:45
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('customer_billing', '0017_summary_renewal_and_future_plan_uuids'),
+        ('customer_billing', '0017_upcoming_invoice_amount_due'),
+    ]
+
+    operations = [
+    ]
```

Comment:
we don't want this merge migration. instead, you should update your 0017 migration to be number 0018 (in the filename) and have a dependency on the *existing* migration 17 `0017_summary_renewal_and_future_plan_uuids`. before you do that, though, you should roll back your local migration state to 0016 `./manage.py migration customer_billing 0016`.